src/Storages/MergeTree/MergeTreeDataSelectExecutor.cpp

```diff
@@ -552,6 +554,31 @@ Pipe MergeTreeDataSelectExecutor::readFromParts(
             useful_indices.emplace_back(index_helper, condition);
     }
 
+    if (settings.force_data_skipping_indices.changed)
+    {
+        std::unordered_set<std::string> useful_indices_names;
+        for (const auto & [ index_helper, _ ]: useful_indices)
+            useful_indices_names.insert(index_helper->index.name);
+
+        std::vector<std::string> forced_indices;
+        boost::split(forced_indices,
```

Comment:
Maybe use full featured parser that will support whitespaces and quoted identifiers?

Reply:
Example:
```
index1, `my index`
```

Reply:
Ok (although looks a bit over kill for this feature to me).